src/public/js/Togethernet/systemMessage.js

```diff
@@ -1,12 +1,57 @@
-export const addSystemMessage = (systemMsg) => {
-  $('#systemMessage').find('span').text(systemMsg);
-  $('#systemMessage').show();
-  $(document).one('mouseup', clearSystemMessage);
-  $(document).one('keydown', clearSystemMessage);
+export const addSystemConfirmMessage = ({
```

Comment:
Based on our discussion, it seems like in most cases you can pass in predefined object with no dynamic properties, without having to pass in these elements individually. So, I think you try something like merging default options, with one of the predefined presets, and then merging with custom options

in src/public/js/Togethernet/systemMessage.js 
```
const defaultSlideStringOptions = {
  msgHeader = '',
  msgBody = '',
  msgFooter = '',
  yayText = '',
  nayText = '',
  yayBtn = 'Continue',
  yayBtnTitle = 'Continue',
  nayBtn = 'Stop',
  nayBtnTitle = 'Stop',
  nayLink = "parent.open('https://togethernet.app')",
}

export const addSystemConfirmMessage = ({defaultOptions, systemMessageOptions = {}, customOptions = {}}) => {
    const messageOptions = {...defaultOptions, ...systemMessageOptions, ...customOptions };
    const {msgHeader, msgBody, msgFooter, yayText...} = messageOptions;
    
      $('#systemConfirmMessage').find('h1').text(msgHeader);
  $('#systemConfirmMessage').find('p').text(msgBody);

...

}

Reply:
@ywenc I'm still a little confused about this. What is the benefit of doing it this way?

Reply:
it would encapsulate all the strings into the system message constants, instead of throwing a lot more lines into classes that should be concerned with other things. Also, it makes things easier to change later on because you would expect the changes to be in where the system messages are, instead of having to hunt them down in many different files.